### PR TITLE
Ability to progressively reveal topics from game state

### PR DESCRIPTION
**Pull request by ptomato**
https://phabricator.endlessm.com/T25677

### CODE REVIEW

File: src/LightSpeed/toolbox.js [hunk 1]
```diff
@@ -18,33 +20,46 @@ var LSToolbox = GObject.registerClass(class LSToolbox extends Toolbox {
 
         this._combinedTopic = new LSCombinedTopic();
         this._combinedTopic.bindGlobal(this._global);
-        this.addTopic('Game', 'rocket-symbolic', this._combinedTopic);
+        this.addTopic('game', 'Game', 'rocket-symbolic', this._combinedTopic);
 
         this._spawnEnemyTopic = new LSUserFunction('spawnEnemy');
         this._spawnEnemyTopic.bindGlobal(this._global);
-        this.addTopic('Enemies', 'astronaut-symbolic', this._spawnEnemyTopic);
+        this.addTopic('spawnEnemy', 'Enemies', 'astronaut-symbolic',
```
Comment (MegFord) [inline src/LightSpeed/toolbox.js:27]: Do you want me to wait to merge this until you have the correct icon for Enemies or do you want to put that in a different PR?

File: src/LightSpeed/toolbox.js [hunk 2]
```diff
@@ -67,4 +82,32 @@ var LSToolbox = GObject.registerClass(class LSToolbox extends Toolbox {
         this._combinedTopic.reset();
         // The user functions are reset by the LightSpeed app already
     }
+
+    async _showTopicsInitially() {
+        const topics = ['spawnEnemy', 'updateAsteroid', 'updateSpinner',
```
Comment (MegFord) [inline src/LightSpeed/toolbox.js:87]: Should 'game' be included here?

File: src/LightSpeed/toolbox.js (hunk 1 above)
Comment (ptomato) [reply src/LightSpeed/toolbox.js:27]: No, I think we can replace the icons separately when the assets are delivered.

File: src/LightSpeed/toolbox.js (hunk 2 above)
Comment (ptomato) [reply src/LightSpeed/toolbox.js:87]: Yeah, good question. I originally didn't include it because that topic is always shown unconditionally, and adding it here would require us to set its corresponding game state key somewhere, otherwise it wouldn't show up. However, that does make it so that we couldn't hide that topic in the future unless we made that change here.

Your call as the reviewer, I guess :smile: I can implement it either way.

Comment (MegFord) [reply src/LightSpeed/toolbox.js:87]: I see! Thanks for the explanation. In that case I'd leave the refactor until it's necessary.
